griptape/processors/base_processors.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod
+from attr import define, field
+from typing import Any
+
+
+@define
+class BasePromptStackProcessor(ABC):
+    pii_replace_text: str = field(default="[PII]", kw_only=True)
```

Comment:
What is the meaning of this default? It looks like it's used in

```python
text.replace(entity["Text"], self.pii_replace_text)
```

which will just swap out allinstances of `entity["Text"]` in the text with `[PII]`. This is not the desired behavior.